internal/pkg/agent/application/upgrade/artifact/download/http/verifier.go

```diff
@@ -194,7 +196,14 @@ func (v *Verifier) composeURI(filename, artifactName string) (string, error) {
 }
 
 func (v *Verifier) getPublicAsc(sourceURI string) ([]byte, error) {
-	resp, err := v.client.Get(sourceURI)
+	ctx, cancelFn := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
I appreciate the usage of a client with a timeout, I am just not sure what triggered this specific change.... :thinking:

Reply:
making linter happy